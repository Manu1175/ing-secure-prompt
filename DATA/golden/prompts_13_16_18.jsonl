{"id": "prompts_13_16_18:0", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "What is the credit score and income of Jane Jones?", "expected_scrub": "What is the credit score and income of <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_13_16_18:1", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "Can you confirm the status of Investment Portfolio for John Miller?", "expected_scrub": "Can you confirm the status of <PRODUCT_NAME> for <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_13_16_18:2", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "What is the masked PIN and biometric method for Michael Lewis?", "expected_scrub": "What is the masked PIN and biometric method for <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_13_16_18:3", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "Provide the latest transaction for IBAN BE75573778827078.", "expected_scrub": "Provide the latest transaction for IBAN <IBAN>.", "entities": null, "notes": null}
{"id": "prompts_13_16_18:4", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "What is the relationship type for Michael Garcia's Credit Card?", "expected_scrub": "What is the relationship type for <CUSTOMER_NAME>'s <PRODUCT_NAME>?", "entities": null, "notes": null}
{"id": "prompts_13_16_18:5", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "What is the credit score and income of Jane Miller?", "expected_scrub": "What is the credit score and income of <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_13_16_18:6", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "Can you confirm the status of Life Insurance for Linda Clark?", "expected_scrub": "Can you confirm the status of <PRODUCT_NAME> for <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_13_16_18:7", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "What is the masked PIN and biometric method for Chris Johnson?", "expected_scrub": "What is the masked PIN and biometric method for <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_13_16_18:8", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "Provide the latest transaction for IBAN BE24699985452067.", "expected_scrub": "Provide the latest transaction for IBAN <IBAN>.", "entities": null, "notes": null}
{"id": "prompts_13_16_18:9", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "What is the relationship type for Emily Johnson's Credit Card?", "expected_scrub": "What is the relationship type for <CUSTOMER_NAME>'s <PRODUCT_NAME>?", "entities": null, "notes": null}
{"id": "prompts_13_16_18:10", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "What is the credit score and income of Robert Williams?", "expected_scrub": "What is the credit score and income of <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_13_16_18:11", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "Can you confirm the status of Savings Account for Katie Miller?", "expected_scrub": "Can you confirm the status of <PRODUCT_NAME> for <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_13_16_18:12", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "What is the masked PIN and biometric method for Jane Brown?", "expected_scrub": "What is the masked PIN and biometric method for <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_13_16_18:13", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "Provide the latest transaction for IBAN BE78676613260743.", "expected_scrub": "Provide the latest transaction for IBAN <IBAN>.", "entities": null, "notes": null}
{"id": "prompts_13_16_18:14", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "What is the relationship type for Jane Clark's Life Insurance?", "expected_scrub": "What is the relationship type for <CUSTOMER_NAME>'s <PRODUCT_NAME>?", "entities": null, "notes": null}
{"id": "prompts_13_16_18:15", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "What is the credit score and income of Anna Smith?", "expected_scrub": "What is the credit score and income of <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_13_16_18:16", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "Can you confirm the status of Personal Loan for Alex Clark?", "expected_scrub": "Can you confirm the status of <PRODUCT_NAME> for <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_13_16_18:17", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "What is the masked PIN and biometric method for Anna Johnson?", "expected_scrub": "What is the masked PIN and biometric method for <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_13_16_18:18", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "Provide the latest transaction for IBAN BE75412829419499.", "expected_scrub": "Provide the latest transaction for IBAN <IBAN>.", "entities": null, "notes": null}
{"id": "prompts_13_16_18:19", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "What is the relationship type for Emily Walker's Life Insurance?", "expected_scrub": "What is the relationship type for <CUSTOMER_NAME>'s <PRODUCT_NAME>?", "entities": null, "notes": null}
{"id": "prompts_13_16_18:0", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "What is the credit score and income of Jane Jones?", "expected_scrub": "What is the credit score and income of <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_13_16_18:1", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "Can you confirm the status of Investment Portfolio for John Miller?", "expected_scrub": "Can you confirm the status of <PRODUCT_NAME> for <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_13_16_18:2", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "What is the masked PIN and biometric method for Michael Lewis?", "expected_scrub": "What is the masked PIN and biometric method for <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_13_16_18:3", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "Provide the latest transaction for IBAN BE75573778827078.", "expected_scrub": "Provide the latest transaction for IBAN <IBAN>.", "entities": null, "notes": null}
{"id": "prompts_13_16_18:4", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "What is the relationship type for Michael Garcia's Credit Card?", "expected_scrub": "What is the relationship type for <CUSTOMER_NAME>'s <PRODUCT_NAME>?", "entities": null, "notes": null}
{"id": "prompts_13_16_18:5", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "What is the credit score and income of Jane Miller?", "expected_scrub": "What is the credit score and income of <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_13_16_18:6", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "Can you confirm the status of Life Insurance for Linda Clark?", "expected_scrub": "Can you confirm the status of <PRODUCT_NAME> for <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_13_16_18:7", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "What is the masked PIN and biometric method for Chris Johnson?", "expected_scrub": "What is the masked PIN and biometric method for <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_13_16_18:8", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "Provide the latest transaction for IBAN BE24699985452067.", "expected_scrub": "Provide the latest transaction for IBAN <IBAN>.", "entities": null, "notes": null}
{"id": "prompts_13_16_18:9", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "What is the relationship type for Emily Johnson's Credit Card?", "expected_scrub": "What is the relationship type for <CUSTOMER_NAME>'s <PRODUCT_NAME>?", "entities": null, "notes": null}
{"id": "prompts_13_16_18:10", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "What is the credit score and income of Robert Williams?", "expected_scrub": "What is the credit score and income of <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_13_16_18:11", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "Can you confirm the status of Savings Account for Katie Miller?", "expected_scrub": "Can you confirm the status of <PRODUCT_NAME> for <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_13_16_18:12", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "What is the masked PIN and biometric method for Jane Brown?", "expected_scrub": "What is the masked PIN and biometric method for <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_13_16_18:13", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "Provide the latest transaction for IBAN BE78676613260743.", "expected_scrub": "Provide the latest transaction for IBAN <IBAN>.", "entities": null, "notes": null}
{"id": "prompts_13_16_18:14", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "What is the relationship type for Jane Clark's Life Insurance?", "expected_scrub": "What is the relationship type for <CUSTOMER_NAME>'s <PRODUCT_NAME>?", "entities": null, "notes": null}
{"id": "prompts_13_16_18:15", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "What is the credit score and income of Anna Smith?", "expected_scrub": "What is the credit score and income of <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_13_16_18:16", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "Can you confirm the status of Personal Loan for Alex Clark?", "expected_scrub": "Can you confirm the status of <PRODUCT_NAME> for <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_13_16_18:17", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "What is the masked PIN and biometric method for Anna Johnson?", "expected_scrub": "What is the masked PIN and biometric method for <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_13_16_18:18", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "Provide the latest transaction for IBAN BE75412829419499.", "expected_scrub": "Provide the latest transaction for IBAN <IBAN>.", "entities": null, "notes": null}
{"id": "prompts_13_16_18:19", "source_path": "external/SecurePrompt/PROMPTS/prompts_13_16_18.xlsx", "modality": "text", "c_level": null, "raw_text": "What is the relationship type for Emily Walker's Life Insurance?", "expected_scrub": "What is the relationship type for <CUSTOMER_NAME>'s <PRODUCT_NAME>?", "entities": null, "notes": null}
