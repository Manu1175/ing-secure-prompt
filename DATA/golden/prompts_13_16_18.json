[
  {
    "text": "What is the credit score and income of Jane Jones?",
    "labels": []
  },
  {
    "text": "Can you confirm the status of Investment Portfolio for John Miller?",
    "labels": []
  },
  {
    "text": "What is the masked PIN and biometric method for Michael Lewis?",
    "labels": []
  },
  {
    "text": "Provide the latest transaction for IBAN BE75573778827078.",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "What is the relationship type for Michael Garcia's Credit Card?",
    "labels": []
  },
  {
    "text": "What is the credit score and income of Jane Miller?",
    "labels": []
  },
  {
    "text": "Can you confirm the status of Life Insurance for Linda Clark?",
    "labels": []
  },
  {
    "text": "What is the masked PIN and biometric method for Chris Johnson?",
    "labels": []
  },
  {
    "text": "Provide the latest transaction for IBAN BE24699985452067.",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "What is the relationship type for Emily Johnson's Credit Card?",
    "labels": []
  },
  {
    "text": "What is the credit score and income of Robert Williams?",
    "labels": []
  },
  {
    "text": "Can you confirm the status of Savings Account for Katie Miller?",
    "labels": []
  },
  {
    "text": "What is the masked PIN and biometric method for Jane Brown?",
    "labels": []
  },
  {
    "text": "Provide the latest transaction for IBAN BE78676613260743.",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "What is the relationship type for Jane Clark's Life Insurance?",
    "labels": []
  },
  {
    "text": "What is the credit score and income of Anna Smith?",
    "labels": []
  },
  {
    "text": "Can you confirm the status of Personal Loan for Alex Clark?",
    "labels": []
  },
  {
    "text": "What is the masked PIN and biometric method for Anna Johnson?",
    "labels": []
  },
  {
    "text": "Provide the latest transaction for IBAN BE75412829419499.",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "What is the relationship type for Emily Walker's Life Insurance?",
    "labels": []
  }
]